apps/db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepository.kt

```diff
@@ -167,6 +169,7 @@ class InntektsmeldingRepository(
                         InntektsmeldingEntitet.id eq innsendingId
                     },
                 ) {
+                    // TODO fjern etter overgangsfase
```

Comment:
Bare for the record: når er egentlig overgangsfasen ferdig? :) 
Er dette en in-flight-overgang, eller noe mer langvarig?

Reply:
Det er snakk om in-flight-overgang, ja.